src/test/regress/expected/multi_mx_call_1.out

```diff
@@ -0,0 +1,337 @@
+-- Test passing off CALL to mx workers
+create schema multi_mx_call;
```

Comment:
can you do the trick similar to here: https://github.com/citusdata/citus/pull/3041/files#diff-3b203d78d59c5fe3d63034e6461ce142R2 

where the output files for not supported versions is minimal?

Reply:
Procedures are supported in PG11 and PG12, but they produce different `CONTEXT` messages if a plpgsql function/procedure raises an exception. That's why I think that it makes sense to have 3 different outputs.

```diff
--CONTEXT:  while executing command on localhost:57638
--PL/pgSQL function multi_mx_call.mx_call_proc_raise(integer) line 4 at RAISE
++CONTEXT:  PL/pgSQL function multi_mx_call.mx_call_proc_raise(integer) line 4 at RAISE
++while executing command on localhost:57638
```

Reply:
Does the following fix it `\set VERBOSITY terse`?

Reply:
ping on this

Reply:
Terse verbosity indeed resolved the difference